streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/config/StreamingConfig.java

```diff
@@ -0,0 +1,25 @@
+package org.ray.streaming.runtime.config;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Streaming config including general, master and worker part.
+ */
+public class StreamingConfig implements Serializable {
+
+  public StreamingMasterConfig masterConfig;
```

Comment:
setter and getter?

Reply:
StreamingConfig will be hold by runtime context with 'private'. No need for setter and getter here.